js/authentication.md

```diff
@@ -549,7 +549,7 @@ this._validAuthStates = ['signedIn'];
 
 Then, in the component's constructor,  implement `showComponent(theme) {}` in lieu of the typical `render() {}` method.
 
-### Federated Identities (Social Sign-in)
+### Federated with Cognito Identities
```

Comment:
Can you update to "Federate with Cognito Identity Pools"

Reply:
Done.